packages/flutter/test/rendering/table_test.dart

```diff
@@ -217,4 +217,29 @@ void main() {
     pumpFrame();
     expect(table, paints..path()..path()..path()..path()..path()..path());
   });
+
+  test('Table flex sizing', () {
+    const BoxConstraints cellConstraints =
+        BoxConstraints.tightFor(width: 100, height: 100);
+    final RenderTable table = RenderTable(
+      textDirection: TextDirection.rtl,
+      children: <List<RenderBox>>[
+        List<RenderBox>.generate(
+          7,
+          (int _) => RenderConstrainedBox(additionalConstraints: cellConstraints),
+        ),
+      ],
+      columnWidths: const <int, TableColumnWidth>{
+        0: FlexColumnWidth(1.0),
+        1: FlexColumnWidth(0.123),
+        2: FlexColumnWidth(0.123),
+        3: FlexColumnWidth(0.123),
+        4: FlexColumnWidth(0.123),
+        5: FlexColumnWidth(0.123),
+        6: FlexColumnWidth(0.123),
+      },
+    );
+
+    layout(table, constraints: BoxConstraints.tight(const Size(800.0, 600.0)));
```

Comment:
I think we probably want to `expect` something about the size after this

Reply:
I added `expect(table.hasSize, true)`.